### DISCUSSION TITLE
Make start of week configurable

### DISCUSSION THREAD

**Discussion by leahoswald** [mealie-recipes/mealie#3287] [Feature Request]
### First Check

- [X] I used the GitHub search to find a similar requests and didn't find it.
- [X] Checked the [tasks tagged](https://github.com/mealie-recipes/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

### Please provide a concise description of the problem that would be addressed by this feature.

I'm from Germany and weeks start here normally at Monday. The calendar date pickers all start at Sunday which confuses me every single time I use them.

### Please provide a concise description of the feature that would resolve your issue.

Using the local settings of the browser or better a profile option to select the start of the week.

### Please consider and list out some caveats or tradeoffs made in your design decision

There shouldn't be such.

### Additional Information

- [ ] If this is accepted I'm willing to submit a PR to provide this feature
- [ ] If this is accepted I'm willing to help maintain this feature
- [ ] I'm willing to sponsor/pay a developer to do this work

**Comment by michael-genson** [comment]
This is already possible via the group settings:

`https://your-url.com/admin/manage/groups` -> click on your group

**Reply by ghuser238490** [reply]
Hi @michael-genson ,
that actually never worked on my instance, the given value (for example 'monday') does not take effect on the date pickers on all pages. The week always starts on sundays. I am able to reproduce it on the demo instance.
Maybe related to
https://github.com/mealie-recipes/mealie/issues/1860
https://github.com/mealie-recipes/mealie/pull/1864

**Reply by RealFoxie** [reply]
I was surprised I would have missed this. Seems like I didn't and I did set it correctly, but the calendar still starts on Sunday.
Might have worked in the past, but not 100% certain.

Testing on Firefox both on Android and pc.

So potentially a bug?

**Reply by leahoswald** [reply]
Same here, I've already set it but it doesn't change the date picker for example here in the meal plan picker it looks like this despite setting the setting to "Monday": 
![Screenshot2024-03-11_20-50-19](https://github.com/mealie-recipes/mealie/assets/1292551/d102bced-5a96-4b43-8f30-e04ea1f58792)

**Reply by michael-genson** [reply]
Apparently there is exactly ONE date picker that it works on, which is why I thought I remembered this working. I've opened a PR to apply this to all date pickers: https://github.com/mealie-recipes/mealie/issues/3292

**Comment by michael-genson** [comment]
Thanks for the details all, and thanks for raising [the issue](https://github.com/mealie-recipes/mealie/issues/3292) @ghuser238490

**Comment by michael-genson** [comment]
This is now resolved with https://github.com/mealie-recipes/mealie/pull/3303
